src/wp-includes/rss.php

```diff
@@ -387,7 +388,7 @@ function map_attrs($k, $v) {
 
 	function error( $errormsg, $lvl = E_USER_WARNING ) {
 		if ( MAGPIE_DEBUG ) {
-			trigger_error( $errormsg, $lvl);
+			wp_trigger_error( __METHOD__, $errormsg, $lvl);
```

Comment:
I'm not sure if adding the method to the error message will be of value in RSS. For now, suggest removing it.

```suggestion
			wp_trigger_error('', $errormsg, $lvl);
```